include/swift/SILOptimizer/Analysis/EpilogueARCAnalysis.h

```diff
@@ -83,8 +83,12 @@ class EpilogueARCContext {
   /// The exit blocks of the function.
   llvm::SmallPtrSet<SILBasicBlock *, 2> ExitBlocks;
 
-  EpilogueARCBlockState &getState(SILBasicBlock *BB) {
-    return IndexToStateMap[*PO->getPONumber(BB)];
+  Optional<EpilogueARCBlockState *> getState(SILBasicBlock *BB) {
```

Comment:
I meant, can you add this in a comment on getState(). A doxygen comment that explains why getState may fail.

Reply:
Done : )